pallets/vanchor/src/lib.rs

```diff
@@ -298,9 +302,9 @@ impl<T: Config<I>, I: 'static> VAnchorInterface<VAnchorConfigration<T, I>> for P
 		// double check the number of roots
 		T::LinkableTree::ensure_max_edges(id, proof_data.roots.len())?;
 		// Check if local root is known
-		T::LinkableTree::ensure_known_root(id, proof_data.roots[0])?;
+		// T::LinkableTree::ensure_known_root(id, proof_data.roots[0])?;
```

Comment:
Why was this commented out?

Reply:
Initially, the tree is empty, so the first transaction will fail, since the leaves, we are making proofs for are not there, which means the root will be zero. I'm thinking about the solution now